app/src/main/java/org/mozilla/focus/utils/FirebaseHelper.java

```diff
@@ -1,97 +1,40 @@
-/* -*- Mode: Java; c-basic-offset: 4; tab-width: 4; indent-tabs-mode: nil; -*-
- * This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
 package org.mozilla.focus.utils;
 
 import android.content.Context;
-import android.support.annotation.NonNull;
-import android.util.Log;
-
-import com.google.android.gms.tasks.OnCompleteListener;
-import com.google.android.gms.tasks.Task;
-import com.google.firebase.remoteconfig.FirebaseRemoteConfig;
-import com.google.firebase.remoteconfig.FirebaseRemoteConfigSettings;
 
 import org.mozilla.focus.BuildConfig;
 
-import java.lang.ref.WeakReference;
 import java.util.HashMap;
 
 /**
- * Provide helper for Firebase functionality
+ * Implementation for FirebaseWrapper.
+ * Also need to provide implementation of getRemoteConfigDefault() method
  */
-final public class FirebaseHelper {
-
+final public class FirebaseHelper extends FirebaseWrapper {
```

Comment:
Helper will be just an implementation to the wrapper 
1. start the wrapper (in a background thread)
2. implement getRemoteConfigDefault to .... provide Remote Config Default